packages/core/src/components/Dialog/__stories__/Dialog.stories.tsx

```diff
@@ -136,7 +136,20 @@ export const Positions = {
             hideTrigger={[]}
             content={
               <DialogContentContainer>
-                <ExampleContent />
+                <Flex
+                  direction="column"
+                  align="start"
+                  gap="small"
+                  style={{ width: "150px", padding: "var(--sb-spacing-small)" }}
+                >
+                  <Skeleton type="text" size="h1" fullWidth />
+                  {Array.from({ length: 3 }, (_value, index: number) => (
+                    <Flex key={index} gap="small" style={{ width: "100%" }}>
+                      <Skeleton type="circle" width={20} height={20} />
```

Comment:
Not for this PR but I think the content of the dialog should be more descriptive rather than a skeleton. Looking at the examples it seems like something is loading in the story